Evergreen/Apps/Get-Minitab.ps1

```diff
@@ -0,0 +1,58 @@
+﻿Function Get-Minitab {
+    <#
+        .SYNOPSIS
+            Get the current version and download URI for the supported releases of Minitab.
+
+        .NOTES
+            Author: Andrew Cooper
+            Twitter: @adotcoop
+    #>
+    [CmdletBinding(SupportsShouldProcess = $False)]
+    [OutputType([System.Management.Automation.PSObject])]
+    param (
+        [Parameter(Mandatory = $False, Position = 0)]
+        [ValidateNotNull()]
+        [System.Management.Automation.PSObject]
+        $res = (Get-FunctionResource -AppName ("$($MyInvocation.MyCommand)".Split("-"))[1]),
+
+        [Parameter(Mandatory = $False, Position = 1)]
+        [ValidateNotNull()]
+        [System.String] $Filter
+    )
+
+    ForEach ($Release in $res.Get.Download.Releases) {
+
+        # Build the update uri based on the release number
+        $uri = $res.Get.Update.Uri -replace $res.Get.Update.ReplaceRelease, $Release
+
+        # Query the update feed
+        $params = @{
+            Uri             = $uri
+            UseBasicParsing = $True
+        }
+        $Updatefeed = (Invoke-WebRequest @params).Content
```

Comment:
Can you use `Invoke-WebRequestWrapper` here? `Invoke-WebRequestWrapper` returns the Content so will achieve what you need here and it handles errors with. `try/catch/throw` block. If `Invoke-WebRequestWrapper` doesn't work, add. `TODO` here and open a seperate issue on what `Invoke-WebRequestWrapper` isn't doing correctly.